### PR TITLE
Fix never expire timings.

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/GeckoPackages/GeckoMemcacheMock/issues/5

### CODE REVIEW

**Comment by SpacePossum** [comment]
@tmatsuo could you check if this works for you?

**Comment by SpacePossum** [comment]
note to self, failing HHVM test is not related

File: src/MemcacheMock/MemcachedMock.php [hunk 1]
```diff
@@ -1245,30 +1256,30 @@ private function isInCache($key)
...
+        return array_key_exists($key, $this->deleteQueue);
+    }
 
-        if ($this->deleteQueue[$key] < time()) {
-            unset($this->deleteQueue[$key]);
+    // @see http://php.net/manual/en/memcached.expiration.php
+    private function normalizeExpirationToAbsoluteTime($expiration)
+    {
+        if (null === $expiration || 0 === $expiration) {
+            return PHP_INT_MAX;
```
Comment (tmatsuo) [inline src/MemcacheMock/MemcachedMock.php]: Why do you need to return PHP_INT_MAX here? Can you just treat `0` as "never expire"?

**Comment by tmatsuo** [comment]
Thanks for the fix!

**Pull request merged**